mp/src/game/client/momentum/ui/spectate/mom_replayui.cpp

```diff
@@ -63,13 +63,12 @@ C_MOMReplayUI::C_MOMReplayUI(IViewPort *pViewport) : Frame(nullptr, PANEL_REPLAY
     m_pGotoTick->SetAllowNumericInputOnly(true);
     m_pGotoTick->AddActionSignalTarget(this);
 
-    m_pTimescaleSlider = new CvarSlider(this, "TimescaleSlider", nullptr, 0.1f, 10.0f, "mom_replay_timescale");
+    m_pTimescaleSlider = new CvarSlider(this, "TimescaleSlider", nullptr, 0.1f, 10.0f, "mom_replay_timescale", false, true);
     m_pTimescaleSlider->AddActionSignalTarget(this);
     m_pTimescaleLabel = new Label(this, "TimescaleLabel", "#MOM_ReplayTimescale");
-    m_pTimescaleEntry = new TextEntry(this, "TimescaleEntry");
+    m_pTimescaleEntry = new CvarTextEntry(this, "TimescaleEntry", "mom_replay_timescale");
```

Comment:
This should use the `%.1f` format as was used in the old `SetLabelText` method.

Reply:
I found a slight bug with how the text entry handles this. If its `%.1f` and you set the cvar value to higher precision (eg. `0.01` then the text entry will read `0.0` and backspace wont do anything.

I think the fix for this would be to set the precision of the float instead of a format override. Say a `SetPrecision(int precision)`. Can get rid of the only integers flag with this too.

Can do this change in the PR thatll change the constructor of the `CvarSlider`and `CvarComboBox` if that sounds good.

Reply:
Yeah that sounds fine. Make the change here so it's not forgotten, then follow up in another branch.